Assets/LeapMotion/Editor/LeapModelPairList.cs

```diff
@@ -0,0 +1,22 @@
+﻿using System.Collections;
+using UnityEditor;
+using UnityEngine;
+
+public static class LeapModelPairList {
+	
+	public static void Show (SerializedProperty list) {
```

Comment:
This looks like it should be a method that belongs in LeapHandPoolEditor, instead of being in a separate class.